docs/APISocketOptions.md

```diff
@@ -877,7 +877,19 @@ In the following cases:
 the connection will be rejected with `SRT_REJ_FILTER` code.
 
 In case of the builtin `fec` filter, the mandatory parameter is `cols`, all
-others have their default values.
+others have their default values. For example, the configuration specified
+as `fec,cols:10` is `fec,rows:1,cols:10,arq:onreq,layout:even`.
+
+Examples for the builtin `fec` filter:
+
+| Peer A               | Peer B      | Negotiated Config            | Result                   |
+|----------------------|-------------|------------------------------|--------------------------|
+| (no filter)          | (no filter) | (no filter)                  | OK                       |
+| fec                  | (no filter) | fec                          | missing `cols` parameter |
+| fec,cols:10          | fec         | fec,cols:10                  | OK                       |
```

Comment:
Negotiated: `fec,cols:10,rows:1,layout:even`

Reply:
I wanted to add an explanation that default values are skipped in this final configuration.

Actually negotiated is: `fec,cols:10,rows:1,layout:even,arq:onreq`.

Reply:
This negotiated value is what is expected in \SRTO_PACKETFILTER\ after a connection is established.

Reply:
Yes, and that's the case. The test application has been added therefore an extra procedure to display that option after connection. Parameters are maybe not in that order, but all parameters are present.